pkgdb2/lib/__init__.py

```diff
@@ -2025,6 +2025,134 @@ def bugzilla(session, name=None):
     return output
 
 
+def _vcs_acls_json(packages, skip_pp=None):
+    """ For a given list of package/user/branch build a dict of dict
+    representating of who has commit access to which package.
+
+    The output dict is something like:
+
+    {
+      pkg1: {
+        branch1: {
+          name: pkg1,
+          branch: branch1,
+          user: [user1, user2],
+          group: [group1, group2]
```

Comment:
No no, the key for groups is singular, see https://github.com/fedora-infra/pkgdb2/pull/166/files#diff-87909695400e927cca78825103e7f028R2139 and above :)